src/hooks/useQuery.ts

```diff
@@ -6,21 +6,33 @@ import { useMemo } from 'react'
  *
  * @param value Input from URL
  */
-function sanitizeInput(value?: string | null): string {
-  return value && Number(value) ? value : '0'
+function sanitizeInput(value?: string | null, defaultValue = '0'): string {
+  return value && Number(value) ? value : defaultValue
 }
 
-export function useQuery(): { sellAmount: string; buyAmount: string } {
+/**
+ * Prevents invalid NEGATIVE numbers from being inserted by hand in the URL
+ *
+ * @param value Input from URL
+ */
+function sanitizeNegativeInput(value?: string | null, defaultValue = '0'): string {
+  return (Number(value) === 0 || Number(value) >= 5) && value ? value : defaultValue
+}
+
+export function useQuery(): { sellAmount: string; buyAmount: string; validUntil?: string } {
   const query = new URLSearchParams(useLocation().search)
 
   // eslint-disable-next-line react-hooks/exhaustive-deps
   const sellAmount = useMemo(() => sanitizeInput(query.get('sell')), [])
   // eslint-disable-next-line react-hooks/exhaustive-deps
   const buyAmount = useMemo(() => sanitizeInput(query.get('buy')), [])
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  const validUntil = useMemo(() => sanitizeNegativeInput(query.get('expires'), '30'), [])
```

Comment:
I'm a bit concerned with empty deps
What if user goes back and forth beteen 
```
https://dex.staging.gnosisdev.com/trade/WETH-USDC?sell=0&buy=0
and
https://dex.staging.gnosisdev.com/trade/USDC-WETH?sell=0&buy=0
```
by just clicking :arrow_up_down: arrows

Reply:
I don't understand this one, what is the issue with the empty deps and switching?

Reply:
`useQuery()` would return the same even if `?sell|buy` changes

Reply:
I will try to reproduce it first

Reply:
#515